kuma/static/js/wiki.js

```diff
@@ -679,4 +679,47 @@
         }
     }
 
+    // listens for post message from interactive editor
+    window.addEventListener('message', function(event) {
+        // get the interactive editor iframe
+        var iframe = document.querySelector('iframe.interactive');
+        var isExpectedOrigin = event.origin === 'https://interactive-examples.mdn.mozilla.net';
+
+        /* there may be other post messages so, ensure that the origin is the
+        expected and, that `event.data` contains an `iframeHeight` property */
+        if (isExpectedOrigin && event.data.iframeHeight) {
+            iframe.setAttribute('height', event.data.iframeHeight);
+        }
+    }, false);
+
+    // Insert notice about permanent language switch
+    var changedLocaleTo = localStorage.getItem('changed-locale-to');
+    if (changedLocaleTo) {
+        var locale = JSON.parse(changedLocaleTo),
+            text = gettext('You are now viewing this article in %(locale_name)s.' +
+                           ' Do you always want to view MDN articles in %(locale_name)s?'),
+            html = interpolate('<div id="locale-permanent-banner" class="primary overheadIndicator"><p>' + text +
+                               '<br><button id="locale-permanent-yes" data-locale="%(locale_code)s">' +
```

Comment:
Yay `<button>`! Button is the right element for this. It also needs the attribute `type=button`.